ray-operator/config/samples/ray-cluster.autoscaler.tls.yaml

```diff
@@ -0,0 +1,414 @@
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-autoscaler
+spec:
+  rayVersion: '2.4.0'
+  # If enableInTreeAutoscaling is true, the autoscaler sidecar will be added to the Ray head pod.
+  # Ray autoscaler integration is supported only for Ray versions >= 1.11.0
+  # Ray autoscaler integration is Beta with KubeRay >= 0.3.0 and Ray >= 2.0.0.
+  enableInTreeAutoscaling: true
+  # autoscalerOptions is an OPTIONAL field specifying configuration overrides for the Ray autoscaler.
+  # The example configuration shown below below represents the DEFAULT values.
+  # (You may delete autoscalerOptions if the defaults are suitable.)
+  autoscalerOptions:
```

Comment:
Is `autoscalerOptions` the only difference compared to `ray-cluster.tls.yaml`? If so, we can add the `autoscalerOptions` section to `ray-cluster.tls.yaml`. I apologize for missing that. I didn't realize the YAML file would be that long.